### ISSUE TITLE
Does not match MATLAB version

### ISSUE THREAD

**Issue by cHemingway** [cHemingway/MRCG_python#1]
The test suite compares against the .mat file generated by the official MATLAB code, and gives the following error:

```
Arrays are not almost equal to 4 decimals
Mismatch: 74.9%
Max absolute difference: 8.99528572
Max relative difference: 57585.18168364
 x: array([[1.1506, 2.0062, 2.6043, ..., 2.4654, 2.378 , 2.4276],
       [2.1931, 2.8548, 3.239 , ..., 3.2363, 3.2794, 3.2412],
       [2.6309, 3.3158, 3.5405, ..., 3.3725, 3.5807, 3.6446],...
 y: array([[1.1506, 2.0062, 2.5889, ..., 2.391 , 2.2481, 2.3641],
       [2.1931, 2.8548, 3.1981, ..., 3.1915, 3.1454, 3.104 ],
       [2.6309, 3.3158, 3.4839, ..., 3.2619, 3.5144, 3.6249],...
----------------------------------------------------------------------
Ran 1 test in 1.774s
```

Looking at the spectrogram, they _look_ quite close, but they are some differences visible as a kind of noise, different per spectrum band, which is understandable as MRCG has scaling for different frequency bands.

![mrcg_comparison](https://user-images.githubusercontent.com/3956920/62553484-547aff80-b867-11e9-8c6b-539a884e944b.png)

**Comment by cHemingway** [comment]
Tracked down to the cochleagram function I think, test output:

```
FAIL: test_value (test_MRCG.Test_all_cochleagrams)
Test all cochleagrams match MATLAB implementation
----------------------------------------------------------------------
[snipped]
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatch: 74.9%
Max absolute difference: 0.96851127
Max relative difference: 0.74390075
 x: array([[1.150633, 2.006189, 2.60432 , ..., 2.465446, 2.377962, 2.427608],
       [2.193056, 2.854786, 3.238984, ..., 3.236343, 3.279402, 3.2412  ],
       [2.630859, 3.315822, 3.540512, ..., 3.372503, 3.580652, 3.644595],...
 y: array([[1.150633, 2.006189, 2.588862, ..., 2.391027, 2.248094, 2.364121],
       [2.193056, 2.854786, 3.198084, ..., 3.191517, 3.145418, 3.104   ],
       [2.630859, 3.315822, 3.483866, ..., 3.261874, 3.514438, 3.624896],...
```

**Comment by cHemingway** [comment]
Closed by 54fa218
